<?xml version="1.0" encoding="utf-8"?>
<launch>
	<!-- start basic args -->
	<arg name="veh" default="$(env VEHICLE_NAME)"/>
	<arg name="config" default="baseline" />
	<arg name="param_file_name" default="default" />
	<arg name="apriltags_param_file_name" default="$(arg param_file_name)" />
	<arg name="local" default="false"/>    
	<!-- end basic args -->
	
	<!-- Start Following -->
	<group>
		<!-- Follow apriltags node -->
		<!-- Receive msg from detect_apriltags_node -->
		<remap from="choose_direction_node/id_info" to="detect_apriltags_node/id_info" />
		<!-- open follow_apriltags_node -->
		<include file="$(find choose_direction_duckie)/launch/choose_direction_node.launch">
			<arg name="veh" value="$(arg veh)"/>
			<arg name="local" value="$(arg local)"/>
		</include-->

	<remap from="inverse_kinematics_node/wheels_cmd" to="wheels_driver_node/wheels_cmd" />
		<remap from="inverse_kinematics_node/car_cmd" to="choose_direction_node/car_cmd"/>
		<!-- Wheel Driver -->
	</group>
</launch>

